test/export/test_torchbind.py

```diff
@@ -120,14 +207,31 @@ def forward(self, x, cc):
                 return x + torch.ops._TorchScriptTesting.takes_foo(cc, x)
 
         cc = torch.classes._TorchScriptTesting._Foo(10, 20)
-        self._test_export_same_as_eager(
-            MyModule(), (torch.ones(2, 3), cc), strict=False
+        ep = self._test_export_same_as_eager(
+            MyModule(), (torch.ones(2, 3), cc), strict=False, pre_dispatch=pre_dispatch
+        )
+        self.assertExpectedInline(
+            ep.module().code.strip(),
+            """\
+def forward(self, arg_0, arg_1):
+    arg1_1, arg2_1, = fx_pytree.tree_flatten_spec(([arg_0, arg_1], {}), self._in_spec)
+    takes_foo_default = torch.ops._TorchScriptTesting.takes_foo.default(arg2_1, arg1_1);  arg2_1 = None
+    add = torch.ops.aten.add.Tensor(arg1_1, takes_foo_default);  arg1_1 = takes_foo_default = None
+    return pytree.tree_unflatten((add,), self._out_spec)""",
         )
-        self._test_export_same_as_eager(
-            MyModule(), (torch.ones(2, 3), cc), strict=False, pre_dispatch=True
+        self.assertExpectedInline(
+            ep.graph_module.code.strip(),
+            """\
+def forward(self, arg0_1, arg1_1, arg2_1):
+    with_effects = torch._higher_order_ops.effects.with_effects(arg0_1, torch.ops._TorchScriptTesting.takes_foo.default, arg2_1, arg1_1);  arg0_1 = arg2_1 = None
+    getitem = with_effects[0]
+    getitem_1 = with_effects[1];  with_effects = None
+    add = torch.ops.aten.add.Tensor(arg1_1, getitem_1);  arg1_1 = getitem_1 = None
+    return (getitem, add)""",  # noqa: B950
         )
 
-    def test_unlift_custom_obj(self):
+    @parametrize("pre_dispatch", [True, False])
```

Comment:
Uh, are the graphs the same for pre_dispatch and not-pre-dispatch?

Reply:
Yeah! For these operators they are the same. We could add an if-else for the assertExpectedInline when necessary.